FrejaEidClient/Client/AuthenticationClient.cs

```diff
@@ -5,6 +5,8 @@
 using Com.Verisec.FrejaEid.FrejaEidClient.Http;
 using Com.Verisec.FrejaEid.FrejaEidClient.Services;
 using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+[assembly: InternalsVisibleTo("FrejaEidClientTest")]
```

Comment:
Why we added this here? If it is needed for tests then we have other classes that have internal constructor and if this is for all of them then, I guess, it should not be here?

Reply:
This is needed for tests, and it used to be in Assembly file, that no longer exists. It can be put in any .cs. Do you recommend moving it to some different file?

Reply:
Maybe to move it to BasicClient? Usually common things go there.

Reply:
Done